src/payment/models/crypto-input/btc-input.service.ts

```diff
@@ -165,14 +165,14 @@ export class BtcInputService extends CryptoInputService {
   }
 
   private async forwardUtxo(input: CryptoInput, address: string): Promise<void> {
-    const outTxId = await this.btcClient.send(
+    const { outTxId, feeAmount } = await this.btcClient.send(
       address,
       input.inTxId,
       input.amount,
       input.vout,
       await this.getFeeRate(input.amount),
     );
-    await this.cryptoInputRepo.update({ id: input.id }, { outTxId });
+    await this.cryptoInputRepo.update({ id: input.id }, { outTxId, forwardFeeAmount: feeAmount });
```

Comment:
I would do the same also for defi-input.service. It's not really important, but brings consistency.